daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/RuntimeData.scala

```diff
@@ -697,7 +698,8 @@ sealed abstract class ErrorERD(local: String, namespaceURI: String)
     null, // fillByteEvArg => FillByteEv
     Nope, // maybeCheckByteAndBitOrderEvArg: => Maybe[CheckByteAndBitOrderEv],
     Nope, // maybeCheckBitOrderAndCharsetEvArg: => Maybe[CheckBitOrderAndCharsetEv],
-    false // isQuasiElementArg: => Boolean
+    false, // isQuasiElementArg: => Boolean
+    null, // runtimeProperties: Map[String,String]
```

Comment:
Even though this is only an `ErrorERD`, we might be safer passing `Map.empty` rather than null so that an `InfosetInputter ` doesn't end up calling `getSimpleText` with null as the second argument.

Reply:
I almost wonder if this is a good thing error? Fields in this ErrorERD should never be used, so having null values would ensure if they were used we could get a NPE and it would be very obvious something was using this when it shouldn't be. If we have valid empty values, we might not realize that some code is using it.

Reply:
Yeah, I guess I'll go with null is ok here as well.